lib/routes/routeBackbeat.js

```diff
@@ -72,6 +73,9 @@ function _isValidLocation(request) {
 PUT /_/backbeat/metadata/<bucket name>/<object key>
 PUT /_/backbeat/data/<bucket name>/<object key>
 PUT /_/backbeat/multiplebackendputobject/<bucket name>/<object key>
+PUT /_/backbeat/multiplebackendinitiatempu/<bucket name>/<object key>
```

Comment:
I would advise to use POST rather than PUT for initiate MPU and complete MPU routes because those are command routes. Also related to this, `multiplebackendputobject` could be named `multiplebackendobject` since we disambiguate with the HTTP action (PUT in this case), but no need to change it since it's not part of this PR.

My personal preference to avoid having too many routes, or form a sort of hierarchy which can help to leverage some common processing code for all routes, would be to use AWS method of having a common route and the specific action in the query string, as e.g. `POST /_/backbeat/multiplebackend/bucket/key?initiatempu`. But at this point I may be a little bit nit-picky :laughing:

Reply:
I like this idea. I've updated this PR with those changes.